index.html

```diff
@@ -197,8 +197,12 @@
           </p>
           <p>
             A HTTP PUT request that includes a <code>Digest</code> header (as described
-            in [[!RFC3230]]) for which the instance-digest in that header does not match
-            the instance it describes, MUST be rejected with a 409 Conflict response.
+            in [[!RFC3230]]) for which any instance-digest in that header does not match the instance
+            it describes, MUST be rejected with a 409 Conflict response.
+          </p>
+          <p>
+            A HTTP PUT/POST request that includes an unsupported <code>Digest</code> type (as described
```

Comment:
The essence of this update is good, but instead of merging `PUT/POST` under the `HTTP PUT 1.4.1` section, I would suggest adding a similar update to the parallel text under HTTP POST 1.3``.